package.json

```diff
@@ -16,6 +16,8 @@
   },
   "dependencies": {
     "expo": "^23.0.0",
+    "lodash": "^4.17.4",
+    "moment": "^2.19.4",
```

Comment:
This is not how monorepo works. You need to update `package.json` in the "hotels" package. Yarn even warn you about this...